src/mpi/init/initthread.c

```diff
@@ -566,7 +566,7 @@ int MPIR_Init_thread(int *argc, char ***argv, int required, int *provided)
 
     /* Create complete request to return in the event of immediately complete
      * operations. Use a SEND request to cover all possible use-cases. */
```

Comment:
```suggestion
      * operations. Use an UNDEFINED request to cover all possible use-cases. */
```

Need to change the comment here too.